### PR TITLE
Add changelog for Dune 3.11

### PR DESCRIPTION
**Pull request by emillon**

### CODE REVIEW

File: data/changelog/dune/2023-09-22-dune-3.11.0.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  # Deprecated and removed
+
+  - Deprecate install destination paths beginning with ".." to prevent packages
+    escaping their designated installation directories. (#8350, @gridbugs)
+
+  - Remove warning against `.dune` files generated by pre dune 2.0 (#8611,
+    @rgrinberg)
+
+  - Remove versions 0.1 and 0.2 of the experimental ctypes extension. (#8293,
+    @emillon)
```
Comment (rikusilvola) [inline data/changelog/dune/2023-09-22-dune-3.11.0.md]:
```suggestion
    @emillon)
---
```

**Comment by tmattio** [comment]
Thanks @emillon! Would you be able to write an announcement as part of the changelog to highlight what's new in the release? It can be short if it's not a significant release

**Comment by emillon** [comment]
@alizter do you want to write a paragraph about the TUI mode?

**Review by christinerose — APPROVED** [review]

**Comment by Alizter** [comment]
```
We are pleased to announce the release of Dune's Terminal User Interface (TUI). Running
dune with `--display tui` will open the TUI. For best results, use in conjunction with watch
mode (`-w`). Here you will see any messages from Dune, allow you to scroll and click to
minimize them. Press `?` to show the help screen. All user feedback is welcome!
```

Something like this?

**Comment by tmattio** [comment]
Thanks for the suggestion @Alizter!

I've added an announcement based on this. @emillon if that looks good to you, this is ready to merge to me.

**Review by christinerose — APPROVED** [review]

**Comment by emillon** [comment]
Yes, looks good to me.

**Comment by emillon** [comment]
Thanks!

**Pull request merged**